### PR TITLE
DEV: ST1K4 needs to be used as a veto device

### PR DESCRIPTION
**Pull request by ghalym**
## Description
Made changes to that ST1K4 encoder value of the Y axis is communicated to the arbiter.
Also, the IN position has been modified based on a request from @tongju12 

## Motivation and Context
TMO needs to Use ST1K4 as a veto devices for it's components.

## Pre-merge checklist
- [x] Code works interactively
- [x] Code contains descriptive comments
- [ ] Test suite passes locally
- [ ] Libraries are set to fixed versions and not ``Always Newest``
- [ ] Code committed with ``pre-commit`` (alternatively ``pre-commit run --all-files``)

### CODE REVIEW

**Comment by ghalym** [comment]
@ZLLentz This is then implemented the same way we do opitcs on the arbiter. And that uses UDINT instead of ULINT. any thoughts?

**Comment by ZLLentz** [comment]
> UDINT instead of ULINT

I assume we were trying to save space on the EL6695 frame size. UDINT gives you a max encoder value of 2^32-1 ~ 4 billion = 4 meters and very few things have greater than 100cm travel range here, so it's sufficient.

ULINT's 64 bits do match the native type, but we don't *need* the counts per se.

It's probably fine either way, we're only saving 4 bytes but it's 4 bytes of zeroes :shrug:

**Review by ZLLentz — APPROVED** [review]
Looks good to me!

File: plc-kfe-motion/kfe_motion/POUs/PRG_ST1K4_TEST.TcPOU [hunk 1]
```diff
@@ -62,7 +65,7 @@ fbMotion(stMotionStage:=Main.M33);
 fbStateSetup(stPositionState:=stDefault, bSetDefault:=TRUE);
 
 fbStateSetup(stPositionState:=astPositionState[E_EpicsInOut.IN], sName:='IN', fPosition:=2);
-fbStateSetup(stPositionState:=astPositionState[E_EpicsInOut.OUT], sName:='OUT', fPosition:=40);
+fbStateSetup(stPositionState:=astPositionState[E_EpicsInOut.OUT], sName:='OUT', fPosition:=40.9);
```
Comment (ZLLentz) [inline plc-kfe-motion/kfe_motion/POUs/PRG_ST1K4_TEST.TcPOU:68]: Can you comment that this happened in the PR description?

**Review by joshc-slac — APPROVED** [review]
too late, but from what I can parse lgtm

**Pull request merged**